rockcraft/services/lifecycle.py

```diff
@@ -60,6 +61,7 @@ def setup(self) -> None:
             package_repositories=project.package_repositories or [],
             project_name=project.name,
             project_vars=project_vars,
+            rootfs_dir=image_info.base_layer_dir,
```

Comment:
@cmatsuoka I had to add this because it looks like the `base_layer_dir` parameter doesn't end up in the ProjectInfo. Can we make that change in craft-parts?

Reply:
Yes, it's a reasonable thing to be there.